src/main/java/gui/MenuController.java

```diff
@@ -139,6 +148,7 @@ public void dragMouse(MouseEvent mouseEvent) {
     public void traverseGraphClicked() {
         int centreNodeID = Integer.parseInt(getNodeTextField().getText());
         drawer.changeZoom(getEndColumn() - getStartColumn(), drawer.getColumnId(centreNodeID));
+        sequenceInfo.setText("Sequence: " + graph.getNode(centreNodeID).getSequence());
```

Comment:
We'll make sure this becomes a relative size or what not eventually. For now it should be fine.